scripts/backtest_uk_gsp.py

```diff
@@ -0,0 +1,467 @@
+"""
+A script to run backtest for PVNet and the summation model for UK regional and national
+
+Use:
+
+- This script uses hydra to construct the config, just like in `run.py`. So you need to make sure
+  that the data config is set up appropriate for the model being run in this script
+- The PVNet and summation model checkpoints; the time range over which to make predictions are made;
+  and the output directory where the results near the top of the script as hard coded user
+  variables. These should be changed.
+
+
+```
+python backtest_uk_gsp.py
+```
+
+"""
+
+try:
+    import torch.multiprocessing as mp
+
+    mp.set_start_method("spawn", force=True)
+    mp.set_sharing_strategy("file_system")
+except RuntimeError:
+    pass
+
+import logging
+import os
+import sys
+
+import hydra
+import numpy as np
+import pandas as pd
+import torch
+import xarray as xr
+from ocf_datapipes.batch import (
+    BatchKey,
+    NumpyBatch,
+    batch_to_tensor,
+    copy_batch_to_device,
+    stack_np_examples_into_batch,
+)
+from ocf_datapipes.config.load import load_yaml_configuration
+from ocf_datapipes.load import OpenGSP
+from ocf_datapipes.training.common import create_t0_and_loc_datapipes
+from ocf_datapipes.training.pvnet import (
+    _get_datapipes_dict,
+    construct_sliced_data_pipeline,
+)
+from ocf_datapipes.utils.consts import ELEVATION_MEAN, ELEVATION_STD
+from omegaconf import DictConfig
+
+# TODO: Having this script rely on pvnet_app sets up a circular dependency. The function
+# `preds_to_dataarray()` should probably be moved here
+from pvnet_app.utils import preds_to_dataarray
+from torch.utils.data import DataLoader
+from torch.utils.data.datapipes.iter import IterableWrapper
+from tqdm import tqdm
+
+from pvnet.load_model import get_model_from_checkpoints
```

Comment:
This may just be me missing something but where are the `load_model` file and the `get_model_from_checkpoints` function referenced in this import? Thought maybe these were meant to be included with this PR?

Reply:
Forgot to add. Done now